# Building 99% Confidence Interval Using Sample Standard Deviation
* given sample size n = 15
* confidencelevel = 0.99
* here we don't have information of population standard diveation so we go for t-distribution

In [8]:
import numpy as np
import scipy.stats as st

In [10]:
data = [1.13, 1.55, 1.43, 0.92, 1.25, 1.36, 1.32, 0.85, 1.07, 1.48, 1.20, 1.33, 1.18, 1.22, 1.29]

In [13]:
interval = st.t.interval(confidence=0.99,df=len(data)-1,loc=np.mean(data),scale=st.sem(data))

In [15]:
np.round(interval,2)

array([1.09, 1.39])

# conclusion
* with 99% confidence A manufacturer of print-heads for personal computers can claim The durability of each print-head (in millions of characters) lies in range [1.09, 1.39] according to t-distribution

# Building 99% Confidence Interval Using Known Population Standard Deviation
* given population standard diveation= 0.2 million charecters
* here we know population standard diveation so we use z-distribution

In [16]:
data = [1.13, 1.55, 1.43, 0.92, 1.25, 1.36, 1.32, 0.85, 1.07, 1.48, 1.20, 1.33, 1.18, 1.22, 1.29]

In [17]:
n=15
std =0.2

In [19]:
interval = st.norm.interval(confidence=0.99,loc=np.mean(data),scale=std/np.sqrt(n))

(1.1056514133957607, 1.3716819199375725)

In [20]:
np.round(interval,2)

array([1.11, 1.37])

# conclusion
* with 99% confidence A manufacturer of print-heads for personal computers can claim The durability of each print-head (in millions of characters) lies in range [1.11, 1.37] according to z distribution